ONSdigital/dp-api-tests
# Dataset api tests (covers dataset, edition, instance and version endpoints)

### What

Title self explainatory

### How to review

Check tests cover functionality for endpoints specified above. Make sure acceptance tests pass for the datatset API, by doing the following: 

1) Clone `dp-dataset-api` repository
2) Checkout `feature/merge-version-into-instance` branch
3) Run `make debug`
4) cd to `dp-api-tests` and run `cd httpexpect/datasetAPI;go test ./...`

### Who can review

Team B
